src/eligibility_matrix.js

```diff
@@ -33,10 +33,14 @@ const RESULTS = {
   not_w2: {label: 'Sorry', md: NOT_SUPPORTED},
   low_earner: {label: 'Sorry', description: 'Please check with your employer.  Lorem ipsum.'},
   
-  high_earner: {label: 'Yay!', description: 'We have more work to do here.  Lorem ipsum.'},
+  high_earner: {label: 'Yay!', description: 'You qualify for Paid Family and Medical Leaave through the California Paid Family Leave program.'},
+  job_protected: {description: 'When you return from your time off, your employer must give you your job back (or an equivalent job.)'},

```

Comment:
In the current code, the `label` is a required item.  However, I think maybe we should just delete `label` entirely, and use only `description`.  What do you think?  If we want to delete `label`, I'm happy to do that.